docs/metrics/customizing-the-sdk/Program.cs

```diff
@@ -70,6 +76,12 @@ public static void Main()
             exponentialBucketHistogram.Record(random.Next(1, 1000), new("tag1", "value1"), new("tag2", "value2"));
         }
 
+        var histogramWithMultipleStream = Meter1.CreateHistogram<long>("HistogramWithMultipleStream");
```

Comment:
nit:

```suggestion
        var histogramWithMultipleStreams = Meter1.CreateHistogram<long>("HistogramWithMultipleStreams");
```

If you take that don't forget to also update the view part. Maybe switch it `instrumentName: "HistogramWithMultipleStream"` -> `instrumentName: histogramWithMultipleStreams.Name` 🤷

Reply:
I'll send a follow up PR, to make entire example use instrumentName.Name instead of hard coding.